.circleci/config.yml

```diff
@@ -1,266 +1,604 @@
+          keys:
+            - data-cache-ds003104
+      - run:
+          name: test ds003104
+          command: |
+            export DS=ds003104
+            python tests/run_tests.py ${DS}
+            mkdir -p ~/reports/${DS}
+            cp ~/mne_data/${DS}/derivatives/mne-bids-pipeline/*/*/*.html ~/reports/${DS}/
+      - store_artifacts:
+          path: /home/circleci/reports/ds003104
+          destination: reports/ds003104
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - mne_data/ds003104/derivatives/mne-bids-pipeline/*/*/*.html
+
+  test_ds003392:
+    <<: *imageconfig
+    steps:
+      - run:
+          <<: *bashenv
+      - run:
+          <<: *xvfb
+      - attach_workspace:
+          at: ~/
+      - restore_cache:
+          keys:
+            - data-cache-ds003392
+      - run:
+          name: test ds003392
+          command: |
+            export DS=ds003392
+            python tests/run_tests.py ${DS}
+            mkdir -p ~/reports/${DS}
+            cp ~/mne_data/${DS}/derivatives/mne-bids-pipeline/*/**/*.html ~/reports/${DS}/
+            cp ~/mne_data/${DS}/derivatives/mne-bids-pipeline/*/**/*.json ~/reports/${DS}/
+            cp ~/mne_data/${DS}/derivatives/mne-bids-pipeline/*/**/*.tsv ~/reports/${DS}/
+      - store_artifacts:
+          path: /home/circleci/reports/ds003392
+          destination: reports/ds003392
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - mne_data/ds003392/derivatives/mne-bids-pipeline/*/*/*.html
+            - mne_data/ds003392/derivatives/mne-bids-pipeline/*/*/*.json
+            - mne_data/ds003392/derivatives/mne-bids-pipeline/*/*/*.tsv
+
+  test_eeg_matchingpennies:
+    <<: *imageconfig
+    steps:
+      - run:
+          <<: *bashenv
+      - run:
+          <<: *xvfb
+      - attach_workspace:
+          at: ~/
+      - restore_cache:
+          keys:
+            - data-cache-eeg_matchingpennies
+      - run:
+          name: test eeg_matchingpennies
+          command: |
+            export DS=eeg_matchingpennies
+            python tests/run_tests.py ${DS}
+            mkdir -p ~/reports/${DS}
+            cp ~/mne_data/${DS}/derivatives/mne-bids-pipeline/*/**/*.html ~/reports/${DS}/
+      - store_artifacts:
+          path: /home/circleci/reports/eeg_matchingpennies
+          destination: reports/eeg_matchingpennies
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - mne_data/eeg_matchingpennies/derivatives/mne-bids-pipeline/*/*/*.html
+
+  build_docs:
+    <<: *imageconfig
+    steps:
+      - run:
+          <<: *bashenv
+      - run:
+          <<: *xvfb
+      - attach_workspace:
+          at: ~/
+      # necessary because gen_examples.py runs config script, which otherwise complains:
+      # ValueError: You set mf_reference_run=01, but your dataset only contains the following runs: [None]
+      - restore_cache:
+          keys:
+            - data-cache-ds000248
```

Comment:
I think this should fix the doc build error. If it does, in principle we could hack in some way *not* to sanity check the `config.py` for this run to get around this at some point, then we don't need to restore the data cache for the doc build, and it can just rely on the HTMLs/reports living in the correct place (which they should based on the `persist_to_workspace` calls made now).